.github/workflows/lint.yml

```diff
@@ -1,13 +1,23 @@
-name: "go-linter"
+name: "Lint"
 
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+    paths:
+      - "**.go"
+      - go.mod
+      - go.sum
+      - .github/workflows/lint.yml
```

Comment:
With these changes, the linter workflow won't run when, say, a Markdown file is edited.